package.json

```diff
@@ -40,9 +40,10 @@
     "README.md"
   ],
   "peerDependencies": {
-    "pixi.js": "^3.0.2"
+    "pixi.js": "^4.0.0"
   },
   "dependencies": {
+    "glslify": "^5.0.2",
     "brfs": "^1.4.1"
```

Comment:
`brfs` is a transform that converts `fs.readFileSync()` calls into the text they would've read in a node environment. I would remove it in favor of glslify as we did in pixi.